packages/cli/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "cli",
```

Comment:
sorry late mentioning this, I think better the package name is `@efishery/eword-cli`, but the folder name is still `cli`

Reply:
oke mas @afrianjunior , updated deui hehe :+1: